src/XeroPHP/Remote/Response.php

```diff
@@ -318,6 +318,7 @@ public function parseXML()
         foreach ($sxml as $child_index => $root_child) {
             switch ($child_index) {
                 case 'PageInfo':
+                case 'pagination':
```

Comment:
@calcinai  So Xero have just done a new release and it's screwed everything up. Instead of PageInfo they've changed it to pagination

"pagination": {
        "page": 1,
        "pageSize": 100,
        "pageCount": 42,
        "itemCount": 4112
    },

<pagination>
        <page>1</page>
        <pageSize>100</pageSize>
        <pageCount>42</pageCount>
        <itemCount>4113</itemCount>
    </pagination>
    
    and using the pageSize parameter works again
?pageSize=1000
<pagination>
        <page>1</page>
        <pageSize>1000</pageSize>
        <pageCount>5</pageCount>
        <itemCount>4113</itemCount>
    </pagination>

Reply:
I can confirm that Xero has again changed their API and that the new pagination value is now handled (I've tested that part).

I have not tested anything relating to the practice manager files but this also looks OK and so does the page size changes.

Reply:
confirmed as well. really not happy with Xero on this one

Reply:
@calcinai any chance of merging and tagging this?